readme.md

```diff
@@ -41,3 +41,5 @@ cleanup documentation in README file
 update tests for new email sending flow
 
 use new logo
+
+update auth flow
```

Comment:
do you think this is the best idea @botany-demo

Reply:
yes i do